### PR TITLE
Home setup

### PR DESCRIPTION
**Pull request by adrian-ponce-wolox**
* Setup project structure
* Add empty home screen with dummy header

captura de pantalla 2017-09-14 a la s 16 19 47

### CODE REVIEW

File: w-books/src/App/components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import wlogo from '../../../assets/Header/wbooks_logo.svg';
+import icon_notifications from '../../../assets/Header/notifications.svg';
+import icon_add_book from '../../../assets/Header/add_book.svg';
+import icon_user from '../../../assets/Header/user_dummy.png';
+
+class Header extends Component {
```
Comment (IgnacioAbadie) [inline w-books/src/App/components/Header/index.js]: if this component hasnt internal states or function to handle, you can declare it like a function.

```
function Header () {
...
..
...
}

export default Header;
```

File: w-books/src/assets/styles.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.Home {
```
Comment (IgnacioAbadie) [inline w-books/src/assets/styles.css]: Take care to create styles only in components folders.
```
Components
        |  Header
        |          | . index.js
        | .        | . styles.js
screens
   | Home
   |        |  index.js
   |        |  layout.js
   |        |  styles.js

   | OtherScreen
   |        |  index.js
   |        |  layout.js
   |        |  styles.js
```
each component have a style file

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]
👍  great! just a little changes

File: w-books/src/App/components/Header/styles.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.Header {
+  background-color: #FFFFFF;
+  box-shadow: 0 5px 5px #DDDDDD;
```
Comment (IgnacioAbadie) [inline w-books/src/App/components/Header/styles.css]: write color like `#DDD` in this cases.

File: w-books/src/App/components/Header/styles.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
+.Header {
```
Comment (IgnacioAbadie) [inline w-books/src/App/components/Header/styles.css:1]: Use flex to styling the components. You shouldnt need use things like positions (relative/absolut)

Comment (IgnacioAbadie) [reply w-books/src/App/components/Header/styles.css:1]: Use flex props like:
```
display: flex
flex-direction: row/column
justify-content: center/flex-start/flex-end/space-between/space-around
align-items: center/flex-start/flex-end
```

File: w-books/src/App/components/Header/styles.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  color: white;
+  height: 96px;
+}
+
+.Header-logo {
+  height: 42px;
+  width: 182px;
+  float: left;
+  position: relative;
+  top: 23px;
```
Comment (IgnacioAbadie) [inline w-books/src/App/components/Header/styles.css]: So, using flex dont need specific position and `top/bottom/left/right`
Elements order by flex properties

File: w-books/src/App/components/Header/styles.css [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+.Header-menu {
+  position: relative;
+  float: right;
+  top: 26px;
+  right: 49px;
+}
+
+.Header-button {
+  margin-left: 15px;
```
Comment (IgnacioAbadie) [inline w-books/src/App/components/Header/styles.css]: margin can be unifiqued by
`margin: 0 15px;`

check this
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

File: w-books/src/App/screens/Home/styles.css [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.Home {
+  text-align: center;
+  margin: 15px;
+  border-color: #DDDDDD;
```
Comment (IgnacioAbadie) [inline w-books/src/App/screens/Home/styles.css]: `#DDD`;

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]

File: w-books/src/App/components/Header/index.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+import icon_add_book from '../../../assets/Header/add_book.svg';
+import icon_user from '../../../assets/Header/user_dummy.png';
+import './styles.css';
+
+function Header () {
+  return (
+    <div className="Header">
+      <img src={wlogo} className="Header-logo" width="182" height="42" alt="WBooks" />
+      <div className="Header-menu">
+        <img className="Header-button notifications" src={icon_notifications} width="30" height="30" alt="Notifications Icon" />
```
Comment (sbalay) [inline w-books/src/App/components/Header/index.js]: Always assign width, height and any ui related property using css

**Review by sbalay — APPROVED** [review]
Great job! Fix @IgnacioAbadie comments and go!

**Comment by adrian-ponce-wolox** [comment]
@IgnacioAbadie Updated the image!

File: w-books/src/App/components/Header/styles.css [hunk 9]
```diff
@@ -0,0 +1,33 @@
+.Header {
+  background-color: #FFFFFF;
```
Comment (IgnacioAbadie) [inline w-books/src/App/components/Header/styles.css]: shorthand color

**Review by IgnacioAbadie — APPROVED** [review]

**Comment by IgnacioAbadie** [comment]
Great!! :smile: Too better!

**Pull request merged**